tests/rhtap-demo/rhtap-demo.go

```diff
@@ -70,11 +70,13 @@ const (
 	jvmRebuildPollingInterval = time.Second * 10
 	snapshotPollingInterval   = time.Second * 1
 	releasePollingInterval    = time.Second * 1
+
+	stageTimeout = time.Minute * 5
 )
 
 var supportedRuntimes = []string{"Dockerfile", "Node.js", "Go", "Quarkus", "Python", "JavaScript", "springboot", "dotnet", "maven"}
 
-var _ = framework.RhtapDemoSuiteDescribe(Label("rhtap-demo"), func() {
+var _ = framework.RhtapDemoSuiteDescribe(Label("rhtap-demo"), Label("verify-stage"), func() {
```

Comment:
why are we adding this label. Isn't it be only `stage` as per https://github.com/redhat-appstudio/infra-deployments/blob/main/.tekton/tasks/verify-smoke-e2e.yaml#L12

Reply:
ok I will change the label in the task itself since this makes more sense.